migration/migrator/data/course_tables.sql

```diff
@@ -570,6 +570,7 @@ CREATE TABLE "threads" (
 	"merged_post_id" int DEFAULT '-1',
 	"is_visible" BOOLEAN NOT NULL,
 	"status" int DEFAULT 0 NOT NULL,
+	"lock_thread_date" timestamp with time zone,
```

Comment:
For changes in the database, migrations are needed. You could follow [migrations#writing-new-migrations](https://submitty.org/developer/migrations#writing-new-migrations).